src/manifests/nginx-deployment.json

```diff
@@ -53,6 +53,13 @@
             },
             "name": "nginx",
             "image": "nginx",
+            "command": [
+              "bash",
+              "-c"
+            ],
+            "args": [
+              "bash /opt/spinnaker/config/default/fetch.sh nginx && nginx -g 'daemon off;'"
+            ],
```

Comment:
We may not need this. The nginx docker image has an entrypoint that starts nginx. I added this to turn on the debug version of nginx